modules/learn/pages/data/data.adoc

```diff
@@ -9,6 +9,8 @@ Couchbase Server saves data as _items_, each of which has a _key_ and a _value_.
 Each item consists of a key and a value.
 Each key is unique within its bucket, and values can be either binary or JSON.
 
+All items are handled by Couchbase Server's xref:learn:services-and-indexes/services/data-service.adoc[Data Service]; and must be encoded as specified in https://tools.ietf.org/html/rfc8259[RFC 8259^].
```

Comment:
So we need to caveat this - only JSON values need to confirm to RFC 8259; binary documents can be whatever the user wants.